susetest/python/susetest.py

```diff
@@ -107,6 +110,10 @@ def __init__(self, name, config):
 		self.ipaddr = self.ipv4_addr
 		self.ip6addr = self.ipv6_addr
 
+                # external ip for cloud
+                self.ipaddr_ext = config.ipv4_ext(self.name)
+                # self.family = os.environ["FAMILY_{}".format(self.name.upper())]
```

Comment:
here, there is a nice variable, that slenkins give: is the env family variable. 
Import testenv 
doesn't support this variable